### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by azimfaiz**
ready

### CODE REVIEW

**Comment by azimfaiz** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
```
Comment (saobv-2223) [inline Gemfile]: image

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: image

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: image

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: image

File: config/credentials.yml.enc [hunk 5]
```diff
@@ -0,0 +1 @@
+B2bY1m7UCdlS02YbEsWxQK9/UCnQ5Mn+9rZNSCGbjhonUjgTtDqXOQnfRNVZaVrgWTf46eCqMA0FTvrLHHAd0netnd+0QLVXnia+p8r4iHKgYLBjVUbmldYjgcZ/swW8VSIQdEpcmF/F3zDNz5c5cHCG+LunNZka2EQoiQDaVRNmqalBU3t/4YsCkVVhZ/L/idPW8IT3cKQ+rQ25V8ptHZquCqdu0+NIRZqezTO2MUJ5dUpponrrPlkM3E+YCWMXsK8HWKv6XD1mSd1mqe368DaK2ucx+gUW1kPXP/8U7kCBRGu/bInCSLhaCiZdKSd/Cb+pVoM1eSv3JnWrPIwEIWx5z8k04jcS4E9aT8J3EaZ91fZb9b+YNMIlPm/MEPAWqph12Rko3n1kk7MBUNXqomIs8UOpgc6r379X--dD+Z02EPVA1vdQ+Q--3oa+4WiJoBdQjexZ1V6HnQ==
```
Comment (saobv-2223) [inline config/credentials.yml.enc]: image

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (saobv-2223) [inline config/database.yml]: image

File: test/channels/application_cable/connection_test.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
+require "test_helper"
```
Comment (saobv-2223) [inline test/channels/application_cable/connection_test.rb]: image

**Comment by azimfaiz** [comment]
ready

File: Gemfile [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+gem "mysql2", "~> 0.5.3"
+gem "puma", "~> 4.1"
+gem "sass-rails", ">= 6"
+gem "webpacker", "~> 4.0"
+gem "turbolinks", "~> 5"
+gem "jbuilder", "~> 2.7"
+
+
```
Comment (manhnt-0960) [inline Gemfile]: remove line 13,14

**Review by manhnt-0960 — COMMENTED** [review]
fix it then merge

**Comment by azimfaiz** [comment]
ready

**Review by saobv-2223 — COMMENTED** [review]
Your PR is having 2 commits, merge to 1 commit please

**Comment by azimfaiz** [comment]
Ready

File: app/javascript/packs/application.js [hunk 9]
```diff
@@ -0,0 +1,8 @@
+
+
+require('@rails/ujs').start()
+require('turbolinks').start()
+require('@rails/activestorage').start()
+require('channels')
+
+
```
Comment (saobv-2223) [inline app/javascript/packs/application.js:8]: remove all the empty lines, please

File: Gemfile [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+ruby "2.7.1"
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+gem "mysql2", "~> 0.5.3"
+gem "puma", "~> 4.1"
+gem "sass-rails", ">= 6"
+gem "webpacker", "~> 4.0"
+gem "turbolinks", "~> 5"
+gem "jbuilder", "~> 2.7"
+gem "bootsnap", ">= 1.4.2", require: false
```
Comment (saobv-2223) [inline Gemfile:13]: image

**Comment by vanvtt-0952** [comment]
merged
plz, merge this pull request

**Pull request merged**